lua/customroles/elementalist/cl_elementalist.lua

```diff
@@ -45,29 +52,25 @@ local function CloseIceOverlay()
 end
 
 local function StartBlindOverlay()
-    darkOverlay = {
-        ["$pp_colour_brightness"] = -1,
-        ["$pp_colour_colour"] = 0
-    }
+    shouldDrawdarkOverlay = true
```

Comment:
This fixes an edge case where the upgraded midnight wouldn't trigger the blind on the first shot, even if it procced